src/Instance/InstanceDetails/InstanceTitleData/AlternativeTitlesList.js

```diff
@@ -0,0 +1,80 @@
+import React, {
+  useMemo,
+} from 'react';
+import PropTypes from 'prop-types';
+import { useIntl } from 'react-intl';
+
+import {
+  MultiColumnList,
+  NoValue,
+} from '@folio/stripes/components';
+
+import {
+  checkIfArrayIsEmpty,
+} from '../../../utils';
+
+const noValue = <NoValue />;
+
+const rowMetadata = ['alternativeTitleTypeId'];
+const visibleColumns = ['type', 'title'];
+const getColumnMapping = intl => ({
+  type: intl.formatMessage({ id: 'ui-inventory.alternativeTitleType' }),
```

Comment:
Respect for not using translations as keys which were the case in the initial implementation.